### ISSUE TITLE
Fix Pages With Multiple H1 Tags

### ISSUE THREAD

**Issue by thejeffreytaylor** [mozilla/bedrock#5869]
Pages with multiple H1 tags are not accessible, as they can't be read in proper order by screen readers. 

The following 3 /En-US pages have multiple H1s, and are listed in order of priority to get fixed ASAP: 

1. https://www.mozilla.org/en-US/privacy/

2. https://www.mozilla.org/en-US/about/manifesto/

3. https://www.mozilla.org/en-US/plugincheck/

These two also need fixing, but have extremely low to no traffic: 

https://www.mozilla.org/en-US/foundation/annualreport/2012/

https://www.mozilla.org/en-US/foundation/annualreport/2012/faq/

**Comment by thejeffreytaylor** [comment]
Adding the blocked label as there's no bandwidth for this for at least a week.

**Comment by shiri1248** [comment]
@thejeffreytaylor  can I work on this issue?
